rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,179 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.4
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka (bukka@php.net), Larry Garfield (larry@garfieldtech.com)
+  * Status: Draft
+
+===== Introduction =====
+
+The Open Source project that develops the PHP language and its reference implementation uses an RFC process to discuss
+and vote on proposed changes. This was created mainly for user facing changes. It works well for that purpose but it is
+not that effective for deciding about purely technical changes that impact PHP internals and extension APIs. It is not
+clear how it should resolve technical conflicts between core developers.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
+php-src when there is a disagreements between core developers.
+
+==== Definitions ====
+
+The text contains various terms that need further clarification to prevent future wording dispute. Here are their
+definitions:
+
+  * php-src - Source code in https://github.com/php/php-src/.
+  * core developer - Anyone with write commit access to php-src.
+  * dev branch - php-src branch consisting of PHP- prefix and major and minor version number (e.g. PHP-8.2).
+  * maintainer - person responsible for core subsystem (e.g. extension or SAPI) that is listed in the php-src EXTENSION file or the php-src OWNERS files if added in the future.
```

Comment:
```suggestion
  * maintainer - person responsible for core subsystem (e.g. extension or SAPI) that is listed in the php-src EXTENSION file or the php-src CODEOWNERS files if added in the future.
```

I assume this is referring to https://github.com/php/php-src/blob/master/CODEOWNERS?

Reply:
Yes of course. Thanks for noticing that.